vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -387,4 +521,11 @@ private void updateContentOrder(String slideId, Integer contentOrder) {
             contentBlockRepository.saveAll(contentBlockList);
         }
     }
+    
+    @Override
+    public void updateVideoTitle(IVideoBlock videoBlock,  String title){
```

Comment:
just make this a save method and pass in the changed block object with the changed title.